iothub/device/src/Transport/Amqp/AmqpAuthenticationRefresher.cs

```diff
@@ -170,5 +57,58 @@ private static string CreateAmqpCbsAudience(IConnectionCredentials connectionCre
             // SAS tokens granted to a group SAS authenticated client will scoped to the IoT hub-level; for example, myHub.azure-devices.net
             return connectionCredentials.HostName;
         }
+
+        async Task<DateTime> IAmqpAuthenticationRefresher.RefreshTokenAsync(CancellationToken cancellationToken)
```

Comment:
Explicit visibility spec.

```suggestion
        internal async Task<DateTime> IAmqpAuthenticationRefresher.RefreshTokenAsync(CancellationToken cancellationToken)
```